content/posts/blog/2024-03-13-image-and-distribution-1-1.md

```diff
@@ -0,0 +1,187 @@
+---
+title: OCI Image and Distribution Specs v1.1 Releases
+author:
+  name: Open Container Initiative
+  tag: oci
+tags: blog
+date: 2024-03-13
+---
+
+The OCI Image Specification and Distribution Specification each had a 1.1.0 release on February 15, 2024.
+These are the first minor releases since the 1.0.0 releases in July 2017 and May 2021 respectively, and include a number of notable changes.
+
```

Comment:
Add a "What's New" section? tl;dr and such
Basically, why are we doing this release? in two specs no less?

Reply:
Also, include a "Clearing the Confusion" around previous dist-spec rc releases wrt "artifacts", since this is a blog post?

Reply:
Also add a section about "Adopters"?
https://kubernetes.io/blog/2023/05/24/oci-security-profiles/
etc

Reply:
Thanks, details added.